cdc/owner/changefeed.go

```diff
@@ -248,7 +248,7 @@ func (c *changefeed) checkStaleCheckpointTs(ctx cdcContext.Context, checkpointTs
 	state := c.state.Info.State
 	if state == model.StateNormal || state == model.StateStopped || state == model.StateError {
 		failpoint.Inject("InjectChangefeedFastFailError", func() error {
-			return cerror.ErrGCTTLExceeded.FastGen("InjectChangefeedFastFailError")
+			return cerror.ErrStartTsBeforeGC.FastGen("InjectChangefeedFastFailError")
```

Comment:
Why did you change the type of error?

Reply:
I removed the ErrGCTTLExceeded error, since the we will fail the changefeed and no need to return a specific error for GCTTL